src/components/Todos/index.js

```diff
@@ -26,20 +50,56 @@ function amendStateWithChildren(DOMSource) {
   };
 }
 
+// THE MAIN FUNCTION
+// This is the implementation of the `main` function that gets passed to
+// Cycle's `run` function.
```

Comment:
We should not assume from this file that this is the `main` given to Cycle.run. That's a decision made only in src/app.js. As far as this file is concerned, it only exports the `Todos` component. This file itself is not opinionated that this component should be the top-level one. That's the point of Cycle Nested: nothing stops you from reusing this `Todos` component inside a larger app.